projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectInformationService.java

```diff
@@ -64,9 +66,35 @@ public Optional<Project> find(ProjectId projectId) {
     return projectRepository.find(projectId);
   }
 
-  public List<OfferIdentifier> queryLinkedOffers(ProjectId projectId) {
-    return projectRepository.find(projectId).map(Project::linkedOffers).orElse(
-        Collections.emptyList());
+  public void updateTitle(String projectId, String newTitle) {
+    ProjectId projectIdentifier = ProjectId.of(UUID.fromString(projectId));
+    Optional<Project> project = projectRepository.find(projectIdentifier);
+    ProjectTitle projectTitle = ProjectTitle.of(newTitle);
+    project.ifPresent(p -> {
+      p.updateTitle(projectTitle);
```

Comment:
We want to fail as early as possible. In case the project title cannot be created (the value object rejects the new input), we want to fail. This should happen independent of whether a project was found or not.